eventlet/wsgi.py

```diff
@@ -456,6 +448,8 @@ def start_response(status, response_headers, exc_info=None):
                 minimum_write_chunk_size = int(self.environ.get(
                     'eventlet.minimum_write_chunk_size', self.minimum_chunk_size))
                 for data in result:
+                    if not isinstance(data, six.binary_type):
```

Comment:
Yes and yes, I'll just make it only fail when there are non-ascii characters (takes care of compatibility issue), GotUnicode is a remain of my earlier patch that I forgot to change, thanks for pointing it out (also once I restore handling unicode strings it'll become ``ValueError`` rather than ``TypeError`` I think)